debian/control

```diff
@@ -103,218 +110,13 @@ Description: Go programming language compiler - metapackage
  This package is a metapackage that, when installed, guarantees
  that (most of) a full Go development environment is installed.
 
-Package: golang-go-linux-amd64
-Architecture: i386 amd64 armel armhf
-Depends: ${shlibs:Depends}, ${misc:Depends}, ${perl:Depends}
-Breaks: golang-go (<< 2:1.1-2)
-Replaces: golang-go (<< 2:1.1-2)
-Description: Go standard library compiled for linux_amd64
- This package contains the Go standard library,
- compiled for linux_amd64.
+Package: golang-pkg
+Architecture: amd64 arm64 armel armhf i386 ppc64 ppc64el
+Depends: ${misc:Depends}, ${perl:Depends}, ${shlibs:Depends}
+Breaks: golang-go (<< 2:1.1-2), golang-go-${go:Hostos}-${go:Hostarch}
+Replaces: golang-go (<< 2:1.1-2), golang-go-${go:Hostos}-${go:Hostarch}
```

Comment:
Yeah, ok, makes sense.  Cross compilation in Go 1.5+ happens entirely without extra stuff needing to be in `GOROOT` (http://dave.cheney.net/2015/03/03/cross-compilation-just-got-a-whole-lot-better-in-go-1-5 is a good article from Dave regarding some of the details).